scoring/templates/scoring/question.html

```diff
@@ -93,7 +91,7 @@ <h3>
         <div id="filter-result" class="row bg-white mb-4 mb-lg-5 mx-auto" style="max-width:100%">
             <div class="d-flex flex-row align-items-center px-3 p-3">
                 <p class="label green mb-0 mr-2">Number of councils</p>
-                <p class="mb-0">XXXX on this group.</p>
+                <p class="mb-0">XXXX councils in this list</p>
```

Comment:
Could do with a `<!-- TODO -->` here to make it more obvious that something’s missing.